packages/schematics/angular/utility/find-module_spec.ts

```diff
@@ -100,5 +108,14 @@ describe('find-module', () => {
       const modPath = findModuleFromOptions(tree, options);
       expect(modPath).toEqual('/projects/my-proj/src/admin/foo.module.ts' as Path);
     });
+
+    it('should find a module using custom ext', () => {
```

Comment:
Same comment.

Reply:
I changed the behavior such that if `moduleExt` is specified, the filename must conform to the ext pattern. So even if a file exists but it does not conform to the pattern, an error will be thrown. Please see the test case below.

Reply:
On second thought, maybe that's bad idea. Instead, it should ignore `moduleExt` if `module` or `module + .ts` exists.